action.yml

```diff
@@ -45,37 +46,47 @@ runs:
     - name: Install
       shell: sh
       working-directory: ${{ inputs.targetFolder }}
-      run: |
-        ${{ inputs.installCommand }}
+      run: ${{ inputs.installCommand }}
 
     - name: Run tests
       shell: sh
       id: run-tests
       working-directory: ${{ inputs.targetFolder }}
       continue-on-error: true
-      run: |
-        ${{ inputs.testCommand }}
+      run: ${{ inputs.testCommand }}
 
-    - name: Find Comment
+    - name: Find comment
       uses: peter-evans/find-comment@v2
+      if: ${{ !env.ACT }}
       id: find-comment
       with:
         issue-number: ${{ github.event.pull_request.number }}
-        comment-author: 'github-action-test-compare[bot]'
+        comment-author: 'github-actions[bot]'
```

Comment:
reverted as author cannot be specified when creating/updating comments, comments are filtered by body includes 'Test compare'

Reply:
can you help me understand this? why would you be able to set one author but not the other? is there anything special about github-actions[bot]?

Reply:
This is the default author for comments created via custom GitHub Actions, I assume a GitHub App would be required to change that behaviour

Reply:
ok that explains it. for the sake of future-proofness, nonetheless, I'd include in the comment some hidden data that we can use and that gives us stronger guarantees that we are finding the comment we're looking for, rather than one that by pure chance contains "test compare" (as unlikely as it may seem)

Reply:
Updated `body-includes` to full comment title